SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -592,6 +592,36 @@ SPATIALGDKEDITOR_API bool GeneratedSchemaFolderExists()
 	return PlatformFile.DirectoryExists(*SchemaOutputPath);
 }
 
+bool DeleteSchemaDatabase()
+{
+	FFileStatData StatData = FPlatformFileManager::Get().GetPlatformFile().GetStatData(*SchemaDatabaseFileName);
+
+	if (StatData.bIsValid)
+	{
+		if (!StatData.bIsReadOnly)
+		{
+			if (!FPlatformFileManager::Get().GetPlatformFile().DeleteFile(*SchemaDatabaseFileName))
+			{
+				// This should never run, since DeleteFile should only return false if the file does not exist which we have already checked for.
+				UE_LOG(LogSpatialGDKSchemaGenerator, Error, TEXT("Unable to delete schema database at %s%s"), *SchemaDatabasePackagePath, *FPackageName::GetAssetPackageExtension());
```

Comment:
```suggestion
				UE_LOG(LogSpatialGDKSchemaGenerator, Error, TEXT("Unable to delete schema database at %s"), *SchemaDatabaseFileName,);
```

Reply:
The existing TryLoadExistingSchemaDatabase() does this `UE_LOG(LogSpatialGDKSchemaGenerator, Error, TEXT("..."), *SchemaDatabasePackagePath, *FPackageName::GetAssetPackageExtension());` rather than `*SchemaDatabaseFileName`. I will change this too unless there is a reason it should be like that.